apps/epic-react-legacy/package.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "epic-react",
+  "name": "epic-react-legacy",
```

we can keep the name epic-react since there won't be a conflict 
```suggestion
  "name": "epic-react",
```